node/tests/k8st/deploy_resources_on_kind_cluster.sh

```diff
@@ -77,9 +77,8 @@ ${kubectl} apply -f $TEST_DIR/infra/calicoctl.yaml
 echo
 
 echo "Wait for Calico to be ready..."
-wait_pod_ready -l k8s-app=calico-node -n calico-system
 wait_pod_ready -l k8s-app=calico-kube-controllers -n calico-system
```

Comment:
We don't need this line any more, since the one below waits for all in the calico-system namespace.